cms-adapter/src/Rsbc.Dmf.CaseManagement/Manager/CaseManager.cs

```diff
@@ -4262,12 +4264,15 @@ public DateTimeOffset GetDpsUplaodDateForERAResults()
                       && x.dfp_dpsclass.Contains("1 - Priority")
                         && (x._dfp_documenttypeid_value == eraResultsGuid || x._dfp_documenttypeid_value == reExamResultsGuid)
                         )
-                    .OrderByDescending(i => i.dfp_uploadeddate)
```

Comment:
good move removing Take(1) but did you intend to remove the sort? Seems useful but I'm not familiar with requirements, marking this resolved

Reply:
wait a minute, am I crazy or will this method always return today? It effectively does nothing. What is the intended purpose of this method? To be fair, I don't think it did anything before you made changes to it.

Reply:
@jburditt  - this code does not look correct.  @shruthi-rapala - it needs to return result, and you also need the OrderByDescending.